src/Components/App.js

```diff
@@ -3,6 +3,7 @@ import React, { useState } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../Logic/calculate';
+import './App.css';
 
 function App() {
   const [data, setData] = useState({
```

Comment:
- [ ] App is converted to a stateful component using class ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))
- [ ] App has state initialized in constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))
- [ ] App has `handleClick` implemented in constructor

```javascript
constructor(props) {
  super(props);
  this.state = {
      total: null,
      next: null,
      operation: null,
  };
  }

  handleClick = (buttonName) => {
  this.setState(calculate(this.state, buttonName));
  }
}
```

- [ ] App passes `handleClick` as a `clickHandler` property to `ButtonPanel` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passHandleClick.png))
- [ ] App passes the `total` or `next` value from state to `Display` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passToatalToDisplay.png))